src/app/screens/Receive/index.tsx

```diff
@@ -232,47 +234,121 @@ function Receive() {
           />
         }
       />
-      {invoice ? (
-        <Container maxWidth="sm">{renderInvoice()}</Container>
-      ) : (
-        <form onSubmit={handleSubmit} className="h-full">
-          <fieldset className="h-full" disabled={loading}>
-            <Container justifyBetween maxWidth="sm">
-              <div className="py-4">
-                <div className="mb-4">
-                  <DualCurrencyField
-                    id="amount"
-                    min={0}
-                    label={t("amount.label")}
-                    placeholder={t("amount.placeholder")}
-                    fiatValue={fiatAmount}
-                    onChange={handleChange}
-                    autoFocus
-                  />
-                </div>
 
+      {isAlbyUser ? (
+        <Container maxWidth="sm">
+          <div className="flex mb-6  mt-4 text-xl ">
+            <Button
```

Comment:
Rather than using two buttons I think we should use a Tabs component to make sure our styling is consistent throughout the app. There is one usage already of Tab.Group in src/app/screens/Home/DefaultView/index.tsx.

@stackingsaunter could you review this issue?

https://github.com/getAlby/lightning-browser-extension/issues/1566

Reply:
I answered in the issue. Definitely we should use one tabs component. If we have tabs component in the extensions, we should use that. Any change to tabs design like discussed in this issue or the design I made for on-chain tab can be updated separately I guess. So either this PR should update tabs component in general or use current one IMHO